app/views/insured/families/_qles_panel.html.erb

```diff
@@ -1,25 +1,36 @@
-<% if EnrollRegistry.feature_enabled?(:show_sep_attestation_options) %>
-  <div class="qles-panel panel panel-default">
-    <div class="panel-heading">
-      <div class="row" id='qle-header'>
-        <div class="col-lg-4 col-md-4 col-sm-12 col-xs-12">
-          <%= image_tag exchange_icon_path('life-event.png'), alt: l10n('.life_event'), style: 'width: 100%' %>
-        </div>
-        <div class="col-lg-8 col-md-8 col-sm-12 col-xs-12 info-wrapper p0">
-          <h4><%= l10n("insured.have_life_changes") %>?</h4>
-          <small>
-            <%= l10n("insured.how_may_effect_your_health_insurance") %>
-          </small>
-        </div>
+<% if @bs4 %>
+  <% if EnrollRegistry.feature_enabled?(:show_sep_attestation_options) %>
+    <div class="home-widget p-2">
+      <h3><%= l10n("insured.report_a_life_change") %></h3>
+      <div class="pr-2 pl-2 pb-2 w-100">
+        <%= h(link_to l10n("insured.update_family_information"), manage_family_insured_families_path(tab: 'family'), class: 'btn w-100 btn-default ml-auto pull-right') %>
```

Comment:
this needs conditional logic per Sarah B. 

> If the user has ever had a draft financial assistance application (regardless of the year), the redirect should be to the applications index.
If the user has never had a financial assistance application, redirect should be to whatever we're renaming the 'Manage Family' page

Reply:
wrapped in faa check https://github.com/ideacrew/enroll/pull/4022/commits/8d2e0ecdcb50420a27185714173553a5e9dc36fd#diff-46a4854b30bb71795971cedfd43979141bf17d1a33a4cb3b5d636e53213ec377R6-R11